features/images/markdown-images.feature

```diff
@@ -21,6 +21,19 @@ Feature: checking embedded Markdown images
       | MESSAGE  | image watermelon.gif |
 
 
+  Scenario: existing local Markdown image with absolute path
+    Given my source code contains the file "documentation/1.md" with content:
+      """
+      ![Alt text](/documentation/images/watermelon.gif "watermelon")
```

Comment:
@ChrisMM this feature spec reproduces the issue you ran into on your Github Intranet PR